### ISSUE TITLE
[Accessibility] [Out logged Page] [AxeDev Tools] Language button has no discernible text

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#858]
**Describe the bug**
The language button has no id or label. The screen reader doesn't identify what's the component for and the user can't change  use the resource provided (in this case, change language).

Ensure buttons have discernible textmore informationLink opens in a new windowElement Location:.z-20<button type="button" tabindex="-1" class="flex block relative z-20 items-center px-4 py-2 bg-white text-sm focus:outline-none ">To solve this problem, you need to fix at least (1) of the following:Element does not have inner text that is visible to screen readersaria-label attribute does not exist or is emptyaria-labelledby attribute does not exist, references elements that do not exist or references elements that are emptyElement has no title attributeElement's default semantics were not overridden with role="none" or role="presentation"

**To Reproduce**
1. Go to 'https://muda.cambiatus.io/join?redirect=%2Fdashboard'
2. Run AxeDev Tools
3. Check the component without label or id

**Expected behavior**
Form components must have id and label to work for screen readers, as 2.5.3 WCAG criteria.

**Screenshots**

**Cambiatus Env you are using**
(https://muda.cambiatus.io/join?redirect=%2Fdashboard)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version 129.0.6668.101 (Versão oficial) 64 bits

**Comment by ceciliabelem** [comment]
For mapping all defect statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
